provisioning/e2e/_service_create_delete.js

```diff
@@ -171,6 +201,9 @@ describe('Provisioning Service Client: CRUD operations', function () {
             debug(err);
           }
           assert.isNull(err,'Should be no error from the create');
+          if (testConfiguration.enrollmentObject.capabilities && testConfiguration.enrollmentObject.capabilities.iotEdge) {
```

Comment:
> && testConfiguration.enrollmentObject.capabilities.iotEdge [](start = 61, length = 59)

Should probably delete everything past the &&

Reply:
Actually I changed my mind.  Leave it exactly the way it is.

---
In reply to: [569003962](https://github.com/Azure/azure-iot-sdk-node/pull/924#discussion_r569003962) [](ancestors = 569003962)